cli/cmd/build/options.go

```diff
@@ -47,4 +41,12 @@ func init() {
 	// Creation time (only for dev purposes)
 	flags.StringVar(&opts.CreatedAt, "created-at", "", "Agent creation time in RFC3339 format")
 	_ = flags.MarkHidden("created-at")
+
+	// Artifacts
+	flags.StringSliceVar(
+		&opts.Artifacts,
+		"artifact",
+		[]string{},
+		"Artifacts to set for the agent. Each artifact should be in the format 'type:url'. Example usage: --artifact type1:url1 --artifact type2:url2",
```

Comment:
the docs for allowed types should also be available here

Reply:
nice catch, updated